src/helpers/chartOptions.ts

```diff
@@ -31,8 +31,11 @@ export interface AreaChartSeries extends ChartSeries {
 } 
 
 export interface MultiSeriesLineChartOptions {
-    connectNulls?: boolean,
+    connectNulls?: boolean
+    showDots? : boolean
```

Comment:
Could we make this something like `dot?: ((props: any) => ReactElement<SVGElement>) | boolean` so that it matches up more closely with the react property.  I have a need to be able to render custom dots.

Reply:
@greinard I would prefer to hold off on the custom dots implementation in this PR to avoid further hold up.

Reply:
Could we at least name it `dot` to match the recharts impl?

At the moment we do a lot of attempted casting of the options before using them.

I think it would be nice if we could move toward just using `{...props.options}` on things like the `Line` component, but that will only work if the property names match.